src/integrations/react.js

```diff
@@ -39,7 +39,7 @@ export function connect(mapStateToProps, actions) {
 					return this.forceUpdate();
 				}
 			};
-			this.componentWillReceiveProps = p => {
+			this.UNSAFE_componentWillReceiveProps = p => {
```

Comment:
Since this change is breaking change, following code may be better. 
```suggestion
			this.componentWillReceiveProps = this.UNSAFE_componentWillReceiveProps = p => {
```